apps/frontend/src/app/page.tsx

```diff
@@ -1,15 +1,96 @@
+'use client';
+import { Plus } from 'lucide-react';
 import Link from 'next/link';
 
+import { CommentSection } from '@/components/comment-section';
+import { DrinkList } from '@/components/drinkAction-drink-list';
+import { EventDiscription } from '@/components/event-discription';
+import { ShareLink } from '@/components/share-link';
+import { Button } from '@/components/ui/button';
+import {
+  Pagination,
+  PaginationContent,
+  PaginationEllipsis,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+} from '@/components/ui/pagination';
+import { UserAvatar } from '@/components/user-avatar';
+import { UserHoverCard } from '@/components/user-hovercard';
 import { events } from '@/models/mockData';
 
-export default function Home() {
+import { LikeButton } from '../../src/components/like-button';
+
+export default function EventFeedPage() {
   return (
-    <main className='flex items-center justify-center'>
-      {events.map((e) => (
-        <Link key={e.id} href={`/events/${e.id}`}>
-          <h2>{e.name}</h2>
+    <main>
+      <div className='flex flex-col items-center justify-center bg-green-300'>
+        <Link href='http://bealcoholic.com'>
```

Comment:
don't hardcode any URLs, you can just use '/' here, that will redirect to the main page